xarray/core/nputils.py

```diff
@@ -18,11 +18,20 @@
 try:
     import bottleneck as bn
 
-    _USE_BOTTLENECK = True
+    _BOTTLENECK_AVAILABLE = True
 except ImportError:
     # use numpy methods instead
     bn = np
-    _USE_BOTTLENECK = False
+    _BOTTLENECK_AVAILABLE = False
+
+try:
+    import numbagg
+
+    _NUMBAGG_AVAILABLE = True
+except ImportError:
+    # use numpy methods instead
+    numbagg = np
+    _NUMBAGG_AVAILABLE = False
```

Comment:
FYI there's a version of this at https://github.com/pydata/xarray/blob/dafd726c36e24ac77427513a4a149a6933353b66/xarray/core/rolling_exp.py#L16-L17, which has the advantage that it contains the version, which we likely want to filter on...

(though I'm not sure my approach is that good! Open to feedback...)